elastic/kibana
# [Infra UI] Centralize asset details components state in a context

closes [#162047](https://github.com/elastic/kibana/issues/162047)
## Summary

This PR refactors the asset details components to use the state context instead of receiving the state via props


### How to test
- Setup a local Kibana instance
- Navigate to `Infrastructure` > `Hosts`
- Open the flyout. Everything should work as before this change
